clikit/src/main/java/org/adamalang/clikit/model/ArgumentDefinition.java

```diff
@@ -0,0 +1,40 @@
+package org.adamalang.clikit.model;
+
+import org.adamalang.clikit.exceptions.XMLFormatException;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import java.util.HashMap;
+
+public class ArgumentDefinition {
+    /** Defines an argument that can be used by CLI commands **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc